default_translations/de/messages.de.yml

```diff
@@ -5,16 +5,24 @@ start:
   registration-text: Du hast noch keinen E-Mail Account bei %domain%? Kein Problem! Wenn eine Freundin oder ein Freund einen Account besitzt, kann diese dir einen Einladungscode zukommen lassen. Mit diesem kannst du dir sofort einen Account registrieren.
   registration-button: Account anlegen
   account-settings: Account verwalten
+  account-settings-desc: Ändere Passwörter und mehr
   account-delete: Account löschen
   aliases: Alias-Adressen
   aliases-delete: Alias-Adresse löschen
+  aliases-desc: Verschleiere deine Identität
   vouchers: Einladungscodes
+  vouchers-desc: Lade Freunde zu diesem Dienst ein
```

Comment:
I would shorten this: "Lade Freund*innen ein"

Reply:
> The invite code infobox looks ugly, they're missing the `class= "alert alert-info"`.

I don't agree. This isn't an alert. Maybe just add a border to this without coloring?

Reply:
You're right, it's not an alert. It's an info box (thus `alert-info`), just like we use it at registration and recovery token. The info box is blue, contrary to the warning (yellow) and error (red) alert boxes.

Reply:
I would reuse this alert box from bootstrap and discuss the coloring/contrast/readability later.